packages/custom-functions-metadata/src/generate.ts

```diff
@@ -148,20 +153,23 @@ export async function generateCustomFunctionsMetadata(
     associate: [],
     errors: [],
   };
+  const fieldsString = "";
 
   if (fs.existsSync(inputFile)) {
     const sourceCode = fs.readFileSync(inputFile, "utf-8");
-    const parseTreeResult: IParseTreeResult = parseTree(sourceCode, inputFile);
-    parseTreeResult.extras.forEach((extra) => extra.errors.forEach((err) => generateResults.errors.push(err)));
+    const parseTreeResultIncludingFields: IParseTreeResultIncludingFields = parseTree(sourceCode, inputFile);
+    parseTreeResultIncludingFields.extras.forEach((extra) => extra.errors.forEach((err) => generateResults.errors.push(err)));
 
     if (generateResults.errors.length > 0) {
       if (wantConsoleOutput) {
         console.error("Errors in file: " + inputFile);
         generateResults.errors.forEach((err) => console.error(err));
       }
     } else {
-      generateResults.metadataJson = JSON.stringify({ functions: parseTreeResult.functions }, null, 4);
-      generateResults.associate = [...parseTreeResult.associate];
+      const metadataObj = convertIParseTreeResultIncludingFieldsToObject(parseTreeResultIncludingFields);
+      generateResults.metadataJson = JSON.stringify(metadataObj, null, 4);
+      // generateResults.metadataJson = JSON.stringify({ functions: parseTreeResultIncludingFields.functions }, null, 4);
```

Comment:
If this comment is not needed, could we remove it?

Reply:
ok

Reply:
> If this comment is not needed, could we remove it?

ok